### DISCUSSION TITLE
Where is 'favicon.ico' after installing rainbowkit using 'yarn create @rainbow-me/rainbowkit'?

### DISCUSSION THREAD

**Discussion by linghuccc** [rainbow-me/rainbowkit#1932] [Q&A]
As title mentioned, I run below command in the installed directory and returned no result:
`find . -type f -name "favicon.ico"`

But in file "pages/index.tsx", it states clearly as below:
```
      <Head>
        <title>RainbowKit App</title>
        <meta
          content="Generated by @rainbow-me/create-rainbowkit"
          name="description"
        />
        <link href="/favicon.ico" rel="icon" />
      </Head>
```

If "favicon.ico" does not exist, why do I still see the next JS triangle as page icon?

Anybody knows? Thank you for your time.

**Comment by 0xmkh** [comment]
@linghuccc We try not include `favicon.ico`. We would rather allow devs to paste that in themselves otherwise if we were to add a `favicon.ico` it'd cache it in the browser for a while. This means if you were to change the favicon it'd take a while to change.
